lib/design/status/failure_page.dart

```diff
@@ -10,10 +10,12 @@ import 'package:hypha_wallet/design/hypha_colors.dart';
 import 'package:hypha_wallet/design/themes/extensions/theme_extension_provider.dart';
 import 'package:hypha_wallet/ui/bottom_navigation/hypha_bottom_navigation.dart';
 
-class SignTransactionFailedPage extends StatelessWidget {
+class FailurePage extends StatelessWidget {
```

Comment:
Why has this class / file been renamed?

Reply:
I initially thought I would use it in a feature unrelated to signing transactions, but I realized that's not the case, so I'll revert it.